### PR TITLE
Megacity blog post

### LINKED ISSUE #75: Blog post about megacity work
https://github.com/whosonfirst-data/whosonfirst-data/issues/1547

This issue was recently completed.. it would be a good candidate for a blog post.

### PR DESCRIPTION
**Pull request by stepps00**
Fixes https://github.com/whosonfirst/whosonfirst-www/issues/75.

### CODE REVIEW

**Comment by stepps00** [comment]
cc @nvkelso - I've added a draft blog post for the recent megacity work, adding you as a reviewer for the draft text.

File: www/blog/2021/02/11/megacities/index.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+## Megacities
+
+Represented by the [`wof:megacity`](https://github.com/whosonfirst/whosonfirst-properties/blob/master/properties/wof/megacity.json) property, these Who’s On First records represent the world's most populated cities. According to [Wikipedia](https://en.wikipedia.org/wiki/Megacity), a "megacity is a very large city, typically with a population of more than 10 million people." Other sources suggest megacities are defined as having populations lower, around 5 million. While Who's On First does not have a population cut-off for how these megacities are defined, we have relied on Natural Earth's [Populated Places](http://www.naturalearthdata.com/downloads/10m-cultural-vectors/) file for this work. Natural Earth identifies 590 global populated places, each identified with a `MEGACITY` attribute, name translations, and concordances.
```
Comment (nvkelso) [inline www/blog/2021/02/11/megacities/index.md]: Let's document the vintage and population cutoff from Natural Earth.

> Natural Earth identifies 590 global populated places, each identified with a `MEGACITY` attribute (actual or emergent), name translations, and concordances, for urban agglomerations with a population of 750,000 or more in 2010.

From:

> Natural Earth identifies 590 global populated places, each identified with a `MEGACITY` attribute, name translations, and concordances.

**Review by nvkelso — APPROVED** [review]
One nit to document how Natural Earth arrived at the 590 count, otherwise LGTM

File: www/blog/2021/02/11/megacities/index.md (hunk 1 above)
Comment (stepps00) [reply www/blog/2021/02/11/megacities/index.md]: addressed in ed9a9fe.

**Comment by stepps00** [comment]
I've tweaked a few sentences, updated two broken links, and given the text another read-through.. this is ready to publish.

Flagging @thisisaaronland - can this be merged to the main branch as-is? Or is there work needed to update additional files?

**Comment by thisisaaronland** [comment]
Sorry for the delay. This didn't pop up in the notifications window. Looking now...

File: www/blog/2021/02/11/megacities/index.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+- Label centroid locations
+- Concordances to Natural Earth and Wikidata, when available
+
+[Here](https://github.com/whosonfirst-data/whosonfirst-data-admin-ir/pull/11/files) is an example of these updates in a GitHub pull request for records in Iran.
+
+#### Updating Geometries
+
+In cases where Who's On First only maintained a `Point` geometry for one of these populated places, we were interested in creating a new `Polygon` geometry. Because many of these localities are represented at more than one placetype in Who's On First - Toronto, for example, is represented as both a [`county`](https://spelunker.whosonfirst.org/id/890457465/), and [`locality`](https://spelunker.whosonfirst.org/id/101735835/) - we were able to use existing "parent" geometries of some of these localities to create (or re-use as) new megacity geometries. In other cases, we hand-curated new geometries using QGIS software.
+
+Because megacities often spread over a larger metropolitan area, we are creating polygons for the “core” cities (which can sometimes be a handful of cities), but we’re exploring creating metropolitan area polygons, too. Please reach out to us if you're interested in those.
```
Comment (thisisaaronland) [inline www/blog/2021/02/11/megacities/index.md]: Please reach out to us should have a link to something, even if it's just the `whosonfirst-data` issues page.

**Comment by thisisaaronland** [comment]
I only had one comment but am happy to approve and deploy now (and update later).

File: www/blog/2021/02/11/megacities/index.md (hunk 2 above)
Comment (stepps00) [reply www/blog/2021/02/11/megacities/index.md]: Resolved in 9c1c7e4

**Comment by stepps00** [comment]
The most recent commit updates links / adds a link to open a new issue in GH... PR should be ready to ship.

**Review by thisisaaronland — APPROVED** [review]

**Pull request merged**